Kitodo/src/main/java/de/sub/goobi/metadaten/Metadaten.java

```diff
@@ -1035,6 +1019,44 @@ private String determineMetadata(DocStruct inStrukturelement, String inTyp) {
     }
 
     /**
+     * Gets metadata value of specific type of an DocStruct element.
+     * @param inStrukturelement
+     *      The DocStruct element.
+     *
+     * @param inTyp
+     *      The metadata typ.
+     * @return
+     *      The metadata value.
+     */
+    public String getMetadataByElementAndType(DocStruct inStrukturelement, String inTyp) {
```

Comment:
Please change parameter names to English-named and there is no need for prefix `in` in front of a parameter name. JavaDoc needs updated afterwards.